src/main/java/uk/gov/hmcts/ccd/domain/model/definition/CriteriaField.java

```diff
@@ -69,4 +70,14 @@ public String getRole() {
     public void setRole(final String role) {
         this.role = role;
     }
+
+    @ApiModelProperty(value = "")
+    @JsonProperty("display_context_parameter")
```

Comment:
Do we need the  @ApiModelProperty(value = "") ?

Reply:
Purely added for consistency with rest of this class.